server/service/src/sync/api/get_site_info.rs

```diff
@@ -0,0 +1,64 @@
+use serde::Deserialize;
+
+use super::*;
+
+#[derive(Debug, PartialEq, Deserialize)]
+pub(crate) struct SiteInfoV5 {
+    pub(crate) id: String,
+    #[serde(rename = "siteId")]
+    pub(crate) site_id: i32,
+    pub(crate) name: String,
+    pub(crate) code: String,
```

Comment:
would suggest only include things that are used (so just id and uuid), technically uuid is not used, but there is a plan to use it, if we allow changing sync details like url and site name

Reply:
Should still do the full json test in mock body though

Reply:
https://github.com/openmsupply/open-msupply/pull/388/commits/96db8d7c803969537fd491f97c2b860548a69678